components/lib/VContainer.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="container">
+    <slot />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'VContainer',
+}
+</script>
+
+<style lang="postcss" scoped>
+.container {
+  max-width: var(--max-container-width);
+  width: var(--container-width);
+  margin: 0 auto;
+}
+@media (max-width: 767px) {
```

Comment:
We'll need a way to remove the hardcoding of the breakpoints from media queries. Possible solutions:

**Using `postcss-media-variables`**

Add this [postcss plugin](https://www.npmjs.com/package/postcss-media-variables) to support variables in media queries. However this is considered non-standard syntax.

```
@media (max-width: var(--breakpoint-mobile) { ... }
```

**Using `postcss-custom-media`**

Add this [postcss plugin](https://www.npmjs.com/package/postcss-custom-media) to support custom media queries.

```
@custom-media --breakpoint-small (max-width: 30em);

@media (--breakpoint-small) {

}
```

Let's looks into these in a separate PR?